src/app/component/CartSummary/CartSummary.componet.js

```diff
@@ -38,6 +38,7 @@ class CartSummary extends Component {
                 subtotal, tax_amount, grand_total, shipping_amount, items
             }
         } = this.props;
+        const cartIsEmpty = Object.keys(items).length === 0;
```

Comment:
There is no need to compare with 0, any value greater than 0 will be counted as true.

Reply:
Also, please separate this line with enter.